### PR TITLE
New Schema for Azure Automation RP

### PR DESCRIPTION
**Pull request by shipram**
Creating a new schema for Azure Automation.
1) The CR is signed off.
2) All tests are passing.
3) Schema validation passing.

### CODE REVIEW

File: schemas/2014-04-01-preview/deploymentTemplate.json [hunk 1]
```diff
@@ -100,26 +120,15 @@
...
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_AAAA"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_CNAME"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_MX"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_NS"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_PTR"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SOA"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SRV"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"},
```
Comment (TianoMS) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: Why are these moved?

Comment (shipram) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: During my merge, these lines got added twice somehow. It came up during my CR and so I removed the duplicates. The lines below the same as the lines being removed.

File: schemas/2015-01-01/deploymentTemplate.json [hunk 2]
```diff
@@ -87,7 +87,7 @@
                   { "$ref": "http://schema.management.azure.com/schemas/2016-02-01-preview/Microsoft.CognitiveServices.json#/resourceDefinitions/accounts" },
                   { "$ref": "http://schema.management.azure.com/schemas/2016-01-29/Microsoft.PowerBI.json#/resourceDefinitions/workspaceCollections" },
                   { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.DataCatalog.json#/resourceDefinitions/catalogs" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.ContainerService.json#/resourceDefinitions/containerServices" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.ContainerService.json#/resourceDefinitions/containerServices" },
```
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: Please remove the extra spaces.

File: schemas/2015-01-01/deploymentTemplate.json [hunk 3]
```diff
@@ -99,26 +99,35 @@
...
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_CNAME"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_MX"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_NS"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_PTR"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SOA"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SRV"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"}
+				  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones" },
```
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: Formatting. Please remove the extra space/tabs.

**Comment by TianoMS** [comment]
Just a few minor issues. The rest looks good.

File: schemas/2015-01-01/deploymentTemplate.json (hunk 3 above)
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: This one doesn't seem to have been fixed.
